src/Cardano/Wallet/Primitive/AddressDiscovery.hs

```diff
@@ -391,13 +391,13 @@ class AddressScheme s where
         -- function as we do not intend to discover any addresses from this
         -- operation; This is merely a lookup from known addresses.
 
-    generateChangeOutput
-        :: Coin
-        -> s
-        -> (TxOut, s)
-        -- ^ Generate a change output 'TxOut' from a given 'Coin'. This picks
-        -- up the first non-used known change address and use it. We keep track
-        -- of pending indexes in the state.
+    nextChangeAddress
+        :: s
+        -> (Address, s)
+        -- ^ Picks the first non-used known change address and use it.
```

Comment:
Nitpick, wouldn't change this now, but believe this ultimately should go on the `SeqState` instance instead of on the class.